docs/concepts/vega-chain/network.md

```diff
@@ -1,27 +1,128 @@
+
+These limits are not totally dissimilar to the spam controls that are also implemented, which aim to mitigate the possibility of malicious actors deliberately flooding the system with junk requests. 
+
+### Maximum pegged orders and liquidity commitment shapes
+Currently, two transaction types can be limited:
+* Total number of pegged orders on all active markets: controlled by network parameter <NetworkParameter frontMatter={frontMatter} param="limits.markets.maxPeggedOrders" />
+* Number of offsets in each liquidity commitment order shape (per side), controlled by <NetworkParameter frontMatter={frontMatter} param="market.liquidityProvision.shapes.maxSize" />
+
+If either parameter's value is decreased (through a governance proposal and vote), then the change does not affect existing orders on the market, but only new orders/liquidity commitments placed after the change is enacted. 
+
+## Spam protection
+On a decentralised and pseudonymous network, there's always a possibility that a malicious actor will attempt to spam blocks and fill them with meaningless transactions. To mitigate that risk, there are spam protections enabled to protect the Vega network, in particular enforced minimums and maximums for transactions sent to the Vega network, and a client-side proof of work requirement to mitigate transaction spam.
+
+The values of all spam protection network parameters can be changed through a governance vote. If a parameter change passes governance, it takes effect in the epoch after it passes.
+
+<!-- link to governance -->
+
+### Proof of work 
+Although Vega is a proof-of-stake network, there is a client-side proof-of-work mechanism to prevent transaction spam from public keys trying to flood the network, or submitting a number of transactions that could slow down the network for all participants. The proof-of-work is calculated by the Vega wallet. It does not incur gas fees and does not have any effect on a transaction's priority.
+
+Every transaction must include a proof-of-work calculation derived from a recent block, proving the transaction was created recently. If a public key submits transactions above the maximum set by the network parameter (<NetworkParameter frontMatter={frontMatter} param="spam.pow.numberOfTxPerBlock" hideValue={true} />), the difficulty of the PoW increases and the proof will take longer to calculate, i.e., transactions will take longer to be generated but can still be included in a block.
+
+It's possible to submit a large volume of transactions at one time, as long as the proof-of-work is calculated by the wallet adequately.
 
-There are several reasons for this decision, including: 
+A transaction with a missing or incorrect proof is rejected, as is any transaction that uses an already-used PoW calculation. A public key that sends transactions with faulty proof-of-work calculations is banned for 30 seconds or 1/48th of an epoch, whichever is greater.
```

Comment:
missing or incorrect proof is rejected but is rejected before a transaction is added to a block so it can't cause a party to be banned.